ontology/yaml/resources/HVAC/entity_types/VAV.yaml

```diff
@@ -472,4 +472,14 @@ VAV_RD_CO2C:
   implements:
   - VAV
   - RD
-  - CO2C
\ No newline at end of file
+  - CO2C
+
+# Additional type for FR-PAR-50A
+VAV_HWZTC_CHWZTC_DTM:
+  description: "Zone temperature control VAV with discharge temperature monitoring and external air volume control (not included)."
+  is_canonical: true
+  implements:
+    - VAV
+    - HWZTC
+    - CHWZTC
```

Comment:
Please confirm: these VAVs have both heating and chilled water coils?

Reply:
Yes I confirm that.